solutions/standard/variables.tf

```diff
@@ -352,19 +352,25 @@ variable "admin_pass_sm_secret_name" {
 ##############################################################
 
 variable "existing_code_engine_project_id" {
-  description = "Existing code engine project ID to deploy Kibana. If no value is passed, a new code engine project will be created."
   type        = string
+  description = "Existing code engine project ID to deploy Kibana. If no value is passed, a new code engine project will be created."
   default     = null
 }
 
 variable "enable_kibana_dashboard" {
   type        = bool
-  description = "Set it true to deploy Kibana in code engine. NOTE: Kibana image is coming direcly from the official registry (https://www.docker.elastic.co/) and not certified by the IBM."
+  description = "Set to true to deploy Kibana in Code Engine. NOTE: By default, the Kibana image will be pulled from the official Elastic registry (docker.elastic.co) and is not certified by IBM, however this can be overridden using the `kibana_registry_namespace_image` and `kibana_image_digest` input."
   default     = false
 }
 
-variable "elasticsearch_full_version" {
-  description = "(Optional) Full version of the Elasticsearch instance in the format `x.x.x` to deploy Kibana dashboard. If no value is passed, data lookup will fetch the full version using the Elasticsearch API, see https://github.com/elastic/kibana?tab=readme-ov-file#version-compatibility-with-elasticsearch"
+variable "kibana_registry_namespace_image" {
+  type        = string
+  description = "Full version of the Elasticsearch instance in the format `x.x.x` to deploy Kibana dashboard. Value is only used if `enable_kibana_dashboard` is true . By default, the image will be fetched from `docker.elastic.co/kibana/kibana`. If overriding this value, ensure it is compatible with your Elasticsearch instance."
+  default     = "docker.elastic.co/kibana/kibana"
+}
+
+variable "kibana_image_digest" {
   type        = string
+  description = "By default, when `enable_kibana_dashboard` is set to true, the DA will deploy a kibana using an image tag version that is applicable with the version of Elasticsearch deployed. Alternatively you can override this by supplying an image digest in the format of `sha256:xxxxx...`, however if doing so ensure the digest entered is an applicable version for your Elasticsearch instance."
```

Comment:
Can you please modify the description as below? 

```
When `enable_kibana_dashboard` is set to true, Kibana is deployed using an image tag compatible with the Elasticsearch version. Alternatively, an image digest in the format `sha256:xxxxx...` can also be specified but it must correspond to a version compatible with the Elasticsearch instance.
```

Reply:
Done.